code/unsupervised/PCA/pca.py

```diff
@@ -0,0 +1,61 @@
+from sklearn.decomposition import PCA
+import matplotlib.pyplot as plt
+import numpy as np
+
+# A value we picked to always display the same results
+# Feel free to change this to view different random value outcomes
+seed = 9000
+
+# We're using a seeded random state so we always get the same outcome
+seeded_state = np.random.RandomState(seed=seed)
+
+# Returns a random 150 points (x, y pairs) in a gaussian distribution,
+# IE most of the points fall close to the average with a few outliers
+rand_points = seeded_state.randn(150, 2)
+
+# The @ operator performs matrix multiplication, and serves to bring
+# our gaussian distribution points closer together
+points = rand_points @ seeded_state.rand(2, 2)
+x = points[:, 0]
+y = points[:, 1]
+
+# Now we have a sample dataset of 150 points to perform PCA on, so
+# go ahead and display this in a plot.
+plt.scatter(x, y, alpha=0.5)
+plt.show()
```

Comment:
i would print out output between the graphs to explain them. I know the code section and comments explain them.